### ISSUE TITLE
Fine tuning XLMRoberta for Question Answering

### ISSUE THREAD

**Issue by houdaM97** [huggingface/transformers#2673]
# ❓ Questions & Help

## Details

**A link to original question on Stack Overflow**: 

I'm trying to fine tune XLM Roberta for Question Answering in tensorflow version, and the question is do i need to convert the pytorch pretrained model to tensorflow because 'xlm-roberta-case' = "https://s3.amazonaws.com/models.huggingface.co/bert/xlm-roberta-base-pytorch_model.bin", if so how can i do it? i tried to use load_pytorch_model_in_tf2_model() but i had errors!!
 thank you for help in advance.

**Comment by LysandreJik** [comment]
Hi, we don't currently have an implementation of XLM RoBERTa in tensorflow.

**Comment by nchocho** [comment]
I guess this one can now be closed since TensorFlow XLM-RoBERTa was released with 2.4.0. Thanks @LysandreJik @jplu . Quick question though: I guess you are not retraining the LM but convert the pytorch weighs. Is there any script in huggingface to do this?

**Comment by LysandreJik** [comment]
Right now the easiest way to convert the PyTorch weights to TensorFlow when the two implementations are in huggingface/transformers is the following, for e.g. XLM-R:

```py
from transformers import XLMRobertaModel, TFXLMRobertaModel

pytorch_model = XLMRobertaModel.from_pretrained("xlm-roberta-base")  # Checkpoint on S3
pytorch_model.save_pretrained("pytorch_checkpoint_directory")  # Save it to a directory

tensorflow_model = TFXLMRobertaModel.from_pretrained("pytorch_checkpoint_directory", from_pt=True)  # Load from directory in TF
```

You can then save that TensorFlow model using the `save_pretrained` method, and you can do it the other way around too to conver TensorFlow models to PyTorch models.

**Comment by LysandreJik** [comment]
@houdaM97, as @nchocho said XLM-R in TensorFlow was released in v2.4.0 last week. There are no official checkpoints on our s3 however, but there are contributed community checkpoints from @jplu you can use instead:

```py
from transformers import TFXLMRobertaModel

model = TFXLMRobertaModel.from_pretrained("jplu/tf-xlm-roberta-base")
```
